foris/locale/de/LC_MESSAGES/foris.po

```diff
@@ -587,10 +589,10 @@ msgid ""
 "all your devices. It usually has less interference, but the signal does not "
 "carry so well indoors."
 msgstr ""
-"Das 2,4 GHz Frequenzband ist unter den Clientgeräten breiter unterstützt, es "
-"ist aber mehr durch Interferenzstörungen belastet. Das 5 GHz Frequenzband "
```

Comment:
I was just curious. I can live with 'Interferenzen'. 'Interferenzstörungen' is kind of more explanatory for me.